packages/core/src/common/_color-aliases.scss

```diff
@@ -7,7 +7,8 @@
 Color aliases
 
 These variables are semantic aliases of our [colors](#colors). They are used throughout Blueprint itself to ensure
-consistent color usage across components.
+consistent color usage across components. Override these variables to change the overall look and feel for your
+application.
```

Comment:
Yeah, I kinda felt that way when writing it. People familiar with SASS should understand that if they see a list of variables, those are overridable.